data/json/items/magazine/9mm.json

```diff
@@ -599,8 +603,9 @@
     "type": "MAGAZINE",
     "name": { "str": "Px4 9x19mm 17-round magazine" },
     "description": "A standard magazine for the Beretta Px4 9x19mm.",
-    "weight": "135 g",
-    "volume": "250 ml",
+    "weight": "90 g",
+    "volume": "78 ml",
+    "longest_side": "122 mm",
```

Comment:
I believe this should have the same length as the 15-round Px4 mag. They look roughly the same: [15-rounder](https://gunmagwarehouse.com/beretta-px4-storm-9mm-15-round-steel-magazine.html) vs. [17-rounder](https://themagshack.com/shop/pistol-magazines/9mm/beretta-px4-storm-9mm-17-round-magazine/). (Granted, it's not a same-scale comparison.) For M9's mags, the 18-rounder is an aftermarket option with a different internal structure; both of these are supposed to be factory magazines.